docs/changelog.md

```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Removed
+- Remove high-level classes because they are implemented in py-sc-kpm
```

Comment:
```suggestion
- Remove high-level classes because they are [implemented in py-sc-kpm](https://github.com/ostis-ai/py-sc-kpm/tree/main/src/sc_kpm)
```